scripts/download_data.py

```diff
@@ -280,14 +293,36 @@ def downloadFastas(self):
                 'display': 'fasta',
                 'range': '{}-{}'.format(0, maxPos)})
             url = '{}%26{}'.format(path, args)
+            tempFileName = '{}.fa.temp'.format(chromosome)
             fileName = '{}.fa'.format(chromosome)
-            downloader = utils.HttpFileDownloader(url, fileName)
+            downloader = utils.HttpFileDownloader(url, tempFileName)
             downloader.download()
+            # We need to replace the header on the downloaded FASTA
+            with open(tempFileName, "r") as inFasta,\
+                    open(fileName, "w") as outFasta:
+                # Write the new header
+                print(">{}".format(chromosome), file=outFasta)
```

Comment:
I suppose it's no big deal... print statements just look ugly to me since they're usually not the right thing to do in systems code.  "print" seems like it should be printing something to the screen, not a file.  This is a script tho so... I guess it's alright.